demos/python_demos/action_recognition/action_recognition_demo/models.py

```diff
@@ -85,14 +85,14 @@ def __init__(self, model_xml, model_bin, ie_core, target_device, num_requests, b
         print("Reading IR...")
         self.net = ie_core.read_network(model_xml, model_bin)
         self.net.batch_size = batch_size
-        assert len(self.net.inputs.keys()) == 1, "One input is expected"
+        assert len(self.net.input_info.keys()) == 1, "One input is expected"
```

Comment:
While you're at it, could you also replace `len(self.net.input_info.keys())` with `len(self.net.input_info)`?

Reply:
Done